core-services/release-controller/_releases/priv/release-ocp-4.11.json

```diff
@@ -239,6 +239,13 @@
                 "name": "periodic-ci-openshift-release-master-ci-4.11-e2e-gcp-techpreview-serial-priv"
             }
         },
+        "hypershift": {
+            "disabled": true,
```

Comment:
Your workflow should be:

1. Edit files in `core-services/release-controller/_releases/`
2. Run `make release-controllers` from the root of the repository

This will automatically generate/update the `priv` files accordingly.

Reply:
Oh I missed that this is the /priv/ file - I actually did what you mentioned and it is not disabled in the other file. Thanks!